test/powershell/Language/Scripting/ActionPreference.Tests.ps1

```diff
@@ -207,6 +207,70 @@ Describe "Tests for (error, warning, etc) action preference" -Tags "CI" {
         $err.FullyQualifiedErrorId | Should -BeExactly "ParameterBindingFailed,Microsoft.PowerShell.Commands.TestPathCommand"
         $err.Exception.InnerException.InnerException | Should -BeOfType "System.Management.Automation.PSInvalidCastException"
     }
+
+    It 'Correctly interprets ErrorActionPreference on Write-Error when it''s reset in a finally block' {
+        $completedCommand = $false
+        $completedFinally = $false
+
+        try
+        {
+            $oldEAP = $ErrorActionPreference
+            $ErrorActionPreference = 'Stop'
```

Comment:
The test name says nothing about "Stop". Make sense to have tests for all possible ErrorActionPreference values?